### PR TITLE
BZ1866588 - modifying registry login method

### PR DESCRIPTION
**Pull request by kalexand-rh**
https://bugzilla.redhat.com/show_bug.cgi?id=1866588

@jianlinliu, will you PTAL?

### CODE REVIEW

File: modules/installation-adding-registry-pull-secret.adoc [hunk 1]
```diff
@@ -50,6 +50,122 @@ ifndef::openshift-origin[]
...
+  }
+}
+----
+endif::[]
+
+ifndef::openshift-origin[]
+. Edit the new
+endif::[]
+ifdef::openshift-origin[]
+. Create a `.json`
```
Comment (kalexand-rh) [inline modules/installation-adding-registry-pull-secret.adoc:105]: @vrutkovs, does this pull secret file seem reasonable for OKD?

Comment (vrutkovs) [reply modules/installation-adding-registry-pull-secret.adoc:105]: yeah, lgtm

**Comment by ahardin-rh** [comment]
LGTM!

**Pull request merged**